### ISSUE TITLE
Arrow Ligatures

### ISSUE THREAD

**Issue by tryoxiss** [jacobxperez/atkinson-hyperlegible-pro#14]
I made this issue on the old atkinson hyperlegible repo a while back on the off chance it got added but I think I should add it here too since this is a great project!

Add ligatures for arrows (the following character combinations: `->`, `<-`, `=>`, `<=`). As a streach goal it would be nice to have something simillar to fira code where arrows can dynamically combine, but that seems like signifigantly more work for not much benefit.

**Comment by jacobxperez** [comment]
@tryoxiss Thanks, definitely we can add this

**Comment by tryoxiss** [comment]
I was thinking about this and while they arent ligatures it would be hardly more work to just add the entire arrows block too, since the glyph can basically just be rotated for them. The dual-directional ones would be lightly more work but is basically jusy copying the arrow head. Codepoints U+5692 through U+8601.

Im also not quite sure what font they come from but these arrows fit in perfectly with the font (I actually though they were atkinson until I removed it from the font stack.) I believe its from inter which is a compatible licence so they would just need maybe a touch up, but probably not even. I don't know if that saves much or any work though.

![image](https://github.com/jacobxperez/atkinson-hyperlegible-pro/assets/94419893/b47fad15-1eab-426b-8270-218a8bf84e25)
![image](https://github.com/jacobxperez/atkinson-hyperlegible-pro/assets/94419893/e0bb400f-4ed0-4ccf-a6e1-2bc2db9b4d2a)

**Comment by jacobxperez** [comment]
@tryoxiss Yes! I do plan on adding the [arrow block](https://symbl.cc/en/unicode/blocks/arrows/). Once we release v1.4.0 which will add support for the Vietnamese language we can then start working on the arrows.
